discretisedfield/field.py

```diff
@@ -3553,29 +3563,29 @@ def from_xarray(cls, xa):
                     "DataArray must have a 'cell' attribute if any "
```

Comment:
Two lines above there is a hard-coded `shape[:3]`. I assume this has to be generalised?